tests/integration/spec.bats

```diff
@@ -81,6 +81,9 @@ function teardown() {
   run bash -c "GOPATH='$GOPATH' go get github.com/xeipuuv/gojsonschema"
   [ "$status" -eq 0 ]
 
+  run git -C "${GOPATH}/src/github.com/xeipuuv/gojsonschema" reset --hard 6637feb73ee44cd4640bb3def285c29774234c7f
```

Comment:
Unit tests should not modify local environments like this. This will lose local changes.

Reply:
But gojsonschema isn't the thing under test, it's a dependency. The same pattern is already used to pin runtime-spec: https://github.com/opencontainers/runc/blob/master/tests/integration/spec.bats#L76

Reply:
This line makes two basically incorrect assumptions that will break:

1. An alternative version of gojsonschema may be in use that could fix this. We can never test with this.
2. A `GOPATH` may consist of multiple paths.

Just because the same pattern is in use doesn't mean that it is correct.